d2cli/export.go

```diff
@@ -0,0 +1,46 @@
+package d2cli
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+type exportExtension string
+
+const GIF exportExtension = ".gif"
+const PNG exportExtension = ".png"
+const PPTX exportExtension = ".pptx"
+const PDF exportExtension = ".pdf"
+const SVG exportExtension = ".svg"
+
+var SUPPORTED_EXTENSIONS = []exportExtension{SVG, PNG, PDF, PPTX, GIF}
+
+func getExportExtension(outputPath string) (exportExtension, error) {
```

Comment:
this is the wrong place to add the check, which is apparent from how often you ignore the errors.

just add a `if ex == ppt, log and return` somewhere in main